icons/norg/norg-plain.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="128" height="px" viewBox="0 0 128 128">
```

Comment:
Height is just `px` here. Not sure if it has any effect, but since we don't know what it does I'd suggest removing it so we don't get unexpected bugs down the line :)

Come to think of it, I think we can remove the width and height property for both icons. Not sure though, but worth trying maybe?

Reply:
Yeah you're right, I just removed the `height="px"` from the plain version

Though removing both width and height properties from both versions seemed to have put both icons in a different size.